### PR TITLE
[SOAR] New Solution Entrust

### PR DESCRIPTION
**Pull request by manishkumar1991**
Required items, please complete.
   
   Change(s):
   - Addition of new soar solution

   Reason for Change(s):
   - New Solution for entrust is needed to make sentinel a competitive product

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

### CODE REVIEW

File: Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIncidentWithIPDetails/readme.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Entrust-Enrich Incident with IP Details Playbook
+ ## Summary
+ When a new Microsoft Sentinel incident is created, this playbook gets triggered and performs below actions
+ 1. Fetches the IP details of his authentication and management activities from Entrust
+ 2. Adds a rich comment to the incident with all the collected information
```
Comment (rahul0216) [inline Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIncidentWithIPDetails/readme.md:5]: Description should be around IP entity, but it is around name entity.

**Review by rahul0216 — COMMENTED** [review]
Address the inline comments. Mostly looks good.

File: Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIP-EntityTrigger/readme.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Entrust-Enrichment of User Details Playbook - entity
+ ## Summary
+ When a Soc analysts, triggered this playbook over some name entity then Playbook performs below actions
+ 1. Fetches the user details, user group and roles details from Entrust
+ 2. Adds a rich comment to the incident associated with entity with all the collected information
```
[the remark this answers is not in the document]
Comment (manishkumar1991) [reply Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIP-EntityTrigger/readme.md:5]: Done

File: Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIncidentWithIPDetails/readme.md (hunk 1 above)
Comment (manishkumar1991) [reply Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIncidentWithIPDetails/readme.md:5]: Done

File: Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-BlockUser/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,2653 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Block Risky/compromized User From Entrust",
```
[the remark this answers is not in the document]
Comment (manishkumar1991) [reply Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-BlockUser/azuredeploy.json]: done

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Pull request merged**